styles.css

```
@@ -0,0 +1,478 @@
+    transition: transform 0.1s ease;
+    box-shadow: 
+        0 4px 8px rgba(0, 188, 212, 0.3),
+        inset 0 2px 4px rgba(255, 255, 255, 0.3);
+    animation: drop-fall linear;
+    z-index: 1000;
+    pointer-events: auto;
+    user-select: none;
+    /* Make sure it stays clickable throughout animation */
+    animation-fill-mode: none;
+    /* Add a subtle pulse to make it more obvious it's clickable */
+    animation: drop-fall linear, pulse 1s ease-in-out infinite alternate;
```

Biome (2.1.2)

[error] Unexpected shorthand property animation after animation-fill-mode



(lint/suspicious/noShorthandPropertyOverrides)